server/server.go

```diff
@@ -363,6 +366,23 @@ func (s *Server) SpeedTestHandler(w http.ResponseWriter, req *http.Request) erro
 	return nil
 }
 
+func (s *Server) SnapshotHandler(w http.ResponseWriter, req *http.Request) error {
+	data, err := s.Store().Save()
+	if err != nil {
+		http.Error(w, "Unexpected snapshot failure", http.StatusInternalServerError)
```

Comment:
The client can print the error to the user and will aid in troubleshooting. We should use a better error message too: failed to create snapshot: +error